render/theinternet.go

```diff
@@ -2,11 +2,50 @@ package render
 
 import (
 	"net"
+	"regexp"
 
 	"github.com/weaveworks/scope/probe/host"
 	"github.com/weaveworks/scope/report"
 )
 
+var (
+	// ServiceNodeIDPrefix is how the ID all service pseudo nodes begin
+	ServiceNodeIDPrefix = "service-"
+
+	knownServicesMatchers = []*regexp.Regexp{
+		// See http://docs.aws.amazon.com/general/latest/gr/rande.html for fainer grained
+		// details
+		regexp.MustCompile(`^.+\.amazonaws\.com$`),
+		regexp.MustCompile(`^.+\.googleapis\.com$`),
+	}
+
+	knownServicesExcluder = []*regexp.Regexp{
+		// We exclude ec2 machines because they are too generic
+		// and having separate nodes for them makes visualizations worse
+		regexp.MustCompile(`^ec2.*\.amazonaws\.com$`),
+	}
+)
```

Comment:
That just ain't going to happen anytime soon. Way too many complications in doing that. It fundamentally changes the nature of the scope app - to date *all* information comes from the probes.

Meanwhile, releasing this feature w/o giving users any way to adjust the matchers/exclusions to their environment is just mean.